docs/modules/ROOT/pages/tmail-backend/features/contactAutocomplete.adoc

```diff
@@ -48,7 +48,89 @@ link:https://github.com/linagora/tmail-lsc#domain-contact-synchronization[domain
 
 == Synchronizing account contacts with third-party applications via RabbitMQ
 
-//TODO
+If you need to synchronize contacts with TMail from an external third-party contact application, you can also use RabbitMQ.
+
+You will need to configure TMail to connect to a RabbitMQ instance that will consume messages sent by the third-party app
+on a dedicated queue.
+
+In your `rabbitmq.properties` file in TMail, you need to add those extra fields:
+
+- `address.contact.uri`: the URI of RabbitMQ having messages related to contacts from the third-party app.
+- `address.contact.user`: the username to access the RabbitMQ instance.
+- `address.contact.password`: the password to access the RabbitMQ instance.
+- `address.contact.exchange`: the exchange name for contacts on RabbitMQ.
```

Comment:
A reminder to sync with Tung work a bit here.
https://github.com/linagora/tmail-backend/pull/405/commits/504768a7c076cdd185a4c475bb5bdcd312ba2985

Reply:
duplicated